gemd/entity/template/has_property_templates.py

```diff
@@ -37,7 +37,7 @@ def properties(self) -> List[Tuple[Union[PropertyTemplate, LinkByUID],
 
         Returns
         -------
-        List[(PropertyTemplate, bounds)]
+        List[(~gemd.entity.template.property_template.PropertyTemplate, BaseBounds)]
```

Comment:
The discrepancies between type hint and docs seem to be present throughout much of this file.